packages/dynamic-pages/src/dynamicPages/generateFiles.js

```diff
@@ -27,8 +38,12 @@ export default new class GenerateFiles {
         // "the request of a dependency is an expression"
         // see: https://github.com/webpack/webpack/issues/196
         //      https://github.com/webpack/webpack/issues/198
-        // eslint-disable-next-line global-require
         this.dynamicRender = require('' + dynamicRenderFile).default;
```

Comment:
Looks like this has been built to accommodate Babel 6/UMD, where we make the assumption that `export default foo` maps to `module.exports.default = foo`. For the sake of versatility, would it make sense to support CommonJS out of the box as well, e.g. `module.exports = foo`, or should we assume users will always utilise `node-babel` or precompile their dynamic render scripts?